### ISSUE TITLE
Fix duplicities and links in js when creating a new talk

### ISSUE THREAD

**Issue by dpordomingo** [src-d/talks#92]
There are JS under:
- `blueprint/per_talk/js/entry.js`
- `blueprint/per_talk/src/js/...`
- `blueprint/shared/js/services/link_track.js`

When it is created a new talk, the ones under `shared` are linked, and the others are copied.
`blueprint/per_talk/js/entry.js` is currently empty, but in the current talks its content is the same, so it may be under `shared`, but maybe not. If it is not, it should be updated with the content that this file has in the current talks.

In summary: the JS stuff should be cleaned, rearranged and documented.

**PS:**  it should be considered #93 while fixing this one

**PS:** Once this has been fixed, review the [CONTRIBUTING.md](https://github.com/src-d/talks/blob/master/CONTRIBUTING.md) because there are links pointing to this issue, and may be changes that should be applyed in the guides
